server/rpc/server.go

```diff
@@ -62,16 +63,18 @@ func NewServer(conf *Config, be *backend.Backend) (*Server, error) {
 		),
 	}
 
-	yorkieServiceCtx, yorkieServiceCancel := context.WithCancel(context.Background())
-	mux := http.NewServeMux()
-	mux.Handle(v1connect.NewYorkieServiceHandler(newYorkieServer(yorkieServiceCtx, be), opts...))
-	mux.Handle(v1connect.NewAdminServiceHandler(newAdminServer(be, tokenManager), opts...))
-	mux.Handle(grpchealth.NewHandler(grpchealth.NewStaticChecker(
+	healthChecker := grpchealth.NewStaticChecker(
 		grpchealth.HealthV1ServiceName,
 		v1connect.YorkieServiceName,
 		v1connect.AdminServiceName,
-	)))
+	)
 
+	yorkieServiceCtx, yorkieServiceCancel := context.WithCancel(context.Background())
+	mux := http.NewServeMux()
+	mux.Handle(v1connect.NewYorkieServiceHandler(newYorkieServer(yorkieServiceCtx, be), opts...))
+	mux.Handle(v1connect.NewAdminServiceHandler(newAdminServer(be, tokenManager), opts...))
+	mux.Handle(grpchealth.NewHandler(healthChecker))
+	mux.Handle(health.NewHTTPHandler(healthChecker))
```

Comment:
How about changing package name to `httphealth` to correspond with `grpchealth` with function name changed to `NewHandler()` from `NewHTTPHandler()`? (Also filename to `httphealth.go` from `health.go`).

Sorry for the repeated change request 🙇🏼

Reply:
I thought it was ugly too
it's a good idea:)